EnglishFilter/sections/specific.txt

```diff
@@ -4117,7 +4117,7 @@ cloudstorageoptions.com##.adslot_left
 ||adee.hit.gemius.pl/gdejs/xgde.js$domain=sportacentrs.com
 ||webgate.tn/getlink/*.js
 ||webgate.tn/promo/*.gif
-tokyoghoulre.com,watchgoblinslayer.com,watchnoblesse.com,watchsnk.net,watchtowerofgod.com,watchdrstone.com,demonslayeranime.com,watchblackclover.com,watchkaguyasama.com,haikyuu3.com,watchoverlord2.com,watchboruto.com,watchsao.tv,watchrezero2.com,techsapparel.com,urlty.in,semawur.com,idzone.site##center > a[href] > img
+tokyoghoulre.com,watchgoblinslayer.com,watchnoblesse.com,watchsnk.net,watchtowerofgod.com,watchdrstone.com,demonslayeranime.com,watchblackclover.com,watchkaguyasama.com,haikyuu3.com,watchoverlord2.com,watchboruto.com,watchsao.tv,watchrezero2.com,techsapparel.com,urlty.in,semawur.com,idzone.site##center > a[href] > img:not([div.textwidget])
```

Comment:
It doesn't work on my end. Try `tokyoghoulre.com,watchgoblinslayer.com,watchnoblesse.com,watchsnk.net,watchtowerofgod.com,watchdrstone.com,demonslayeranime.com,watchblackclover.com,watchkaguyasama.com,haikyuu3.com,watchoverlord2.com,watchboruto.com,watchsao.tv,watchrezero2.com,techsapparel.com,urlty.in,semawur.com,idzone.site##center > a[href] > img:not([class^="alignnone"])`